localisation/00_zuwc_united_localisation.csv

```diff
@@ -2303,21 +2292,21 @@ text_effect_extra_soul_shard;Has chance to get 
 EVTOPTB_LT_60001_evil;Kill them and take their possessions!;;T�tet sie und nehmt ihre Besitzt�mer!;;;;;;;;;;;x
-RULE_HOLY_WARS;�RWarcraft�!: Holy Wars;�RWarcraft�!: Guerres saintes;�RWarcraft�!: Heilige Kriege;;;;;;;;;;;x
+RULE_HOLY_WARS;�RWarCraft�!: Holy Wars;�RWarCraft�!: Guerres saintes;�RWarCraft�!: Heilige Kriege;;;;;;;;;;;x
 RULE_HOLY_WARS_RESTRICTED;Restricted;Avec restriction;Eingeschr�nkt;;;;;;;;;;;x
-RULE_HOLY_WARS_RESTRICTED_DESC;Holy War' CB is available only for characters with 'Zealous' trait and aggressive religions like the Scarlet Light and Shath'gral.;Les guerres saintes ne sont activ�es que pour pour les personnage ayant le trait 'Z�l�' et une religion belliqueuse comme la Lumi�re �carlate et les Shath'gral.;Der heilige Krieg CB ist nur f�r eifrige Charaktere und aggressive Religionen wie Scarlet Light und Shath'gral verf�gbar.;;;;;;;;;;;x
+RULE_HOLY_WARS_RESTRICTED_DESC;Holy War' CB is available only for characters with "Zealous" trait and aggressive religions like the Scarlet Light and Shath'gral.;Les guerres saintes ne sont activ�es que pour pour les personnage ayant le trait 'Z�l�' et une religion belliqueuse comme la Lumi�re �carlate et les Shath'gral.;Der heilige Krieg CB ist nur f�r eifrige Charaktere und aggressive Religionen wie Scarlet Light und Shath'gral verf�gbar.;;;;;;;;;;;x
 RULE_HOLY_WARS_ON;On;R�gle activ�e;Aktiviert;;;;;;;;;;;x
 RULE_HOLY_WARS_ON_DESC;Holy War' CB is available for all.;Les guerres saintes sont activ�es pour tous.;Der heilige Krieg CB ist f�r alle verf�gbar.;;;;;;;;;;;x
 RULE_HOLY_WARS_OFF;Off;R�gle d�sactiv�e;Deaktiviert;;;;;;;;;;;x
 RULE_HOLY_WARS_OFF_DESC;Holy War' CB is disabled.;Les guerres saintes sont d�sactiv�es.;Der heilige Krieg CB ist deaktiviert.;;;;;;;;;;;x
 kolk;Blood of [This.GetFirstNameWithNick];;Blut von [This.GetFirstNameWithNick];;;;;;;;;;;x
-kolk_desc;Kolk was one of the first centaur, directly descended from the union of Therazane and Zaetar. He was the First Khan as well as progenitor of the Kolkar.;;Kolk war einer der ersten Zentauren, der direkt aus der Verbindung von Therazane und Zaetar hervorging. Er war der erste Khan und auch der Stammvater der Kolkar.;;;;;;;;;;;x
+kolk_desc;Kolk was one of the first centaurs, directly descended from the union of Therazane and Zaetar. He was the first Khan as well as progenitor of the Kolkar.;;Kolk war einer der ersten Zentauren, der direkt aus der Verbindung von Therazane und Zaetar hervorging. Er war der erste Khan und auch der Stammvater der Kolkar.;;;;;;;;;;;x
```

Comment:
**_First Khan_** is a title.
https://wow.gamepedia.com/Kolk

Reply:
Ah, ok. I just thought of him being the first one, not of it as a title made of both words. done